src/common/thread_measurement.cpp

```diff
@@ -5,13 +5,12 @@
 #include "cmsis_os.h" //osDelay
 #include "filament_sensor.hpp"
 #include "marlin_client.h"
+#include "marlin_server.h"
 #include "trinamic.h"
 #include "stm32f4xx_hal.h"
+#include "eeprom.h"
 
-static inline bool checkTimestampsAscendingOrder(uint32_t a, uint32_t b) {
-    uint32_t u = (b - a);
-    return !(u & 0x80000000u);
-}
+static bool current_detect_filament_insert = false;
```

Comment:
/question/ Why isn't this a local variable? I don't see it being used anywhere else then in StartMeasurementTask().

Reply:
Solved